src/platform/packages/shared/kbn-esql-ast/src/commands_registry/commands/row/autocomplete.ts

```diff
@@ -28,7 +28,11 @@ export async function autocomplete(
   const innerText = query.substring(0, cursorPosition);
   // ROW col0 = /
   if (/=\s*$/.test(innerText)) {
-    return getFunctionSuggestions({ location: Location.ROW }, callbacks?.hasMinimumLicenseRequired);
+    return getFunctionSuggestions(
```

Comment:
honestly I would like to pass the whole callbacks to this function too.  
But I don't see any context or callbacks in the module where it is implemented